SimG4Core/Application/src/ParametrisedEMPhysics.cc

```diff
@@ -32,20 +32,30 @@
 #include "G4AntiProton.hh"
 
 #include "G4EmParameters.hh"
-#include "G4EmProcessOptions.hh"
 #include "G4PhysicsListHelper.hh"
 #include "G4SystemOfUnits.hh"
 #include "G4UAtomicDeexcitation.hh"
 #include "G4LossTableManager.hh"
+#include "G4ProcessManager.hh"
+#include "G4Transportation.hh"
+
+G4ThreadLocal std::unique_ptr<GFlashEMShowerModel> ParametrisedEMPhysics::theEcalEMShowerModel;
```

Comment:
@civanch could you please clarify for my understanding why these variables are redefined here if they are already [resent as static members in the header?

Reply:
@civanch ping...

Reply:
@fabiocos , I am not sure if this line is the redefinition, may be it can be removed. I can check in the next PR.

Reply:
@civanch we have still time before closing 10_3_X, is not possible to clarify it now?